maquetado/scss/detail.scss

```diff
@@ -0,0 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+%equal-font-heights {
+ 
+  font-family: Roboto, sans-serif;;
+  display: flex;
+  align-items:baseline;
+
+}
+
+$color-1: #828282;
+$color-2: #000;
+$color-3: #BED23C;
+$color-4: #FFF;
```

Comment:
For the variable names, we usually use https://chir.ag/projects/name-that-color/#6195ED so we know what color it is. Think of SASS variables the same as any other language, the naming must be self-explanatory